doc/compressed_transactions.md

```diff
@@ -0,0 +1,49 @@
+# Compressed Transaction Schema
+
+## 1. Abstraction
+With this transaction compression schema we reduce the size of all transactions by at least 15-20% and up to 40% with signature recovery. The main current uses for this will be steganography or other low bandwidth channels for passing transaction data.
```

Comment:
In 61504d1a4d7ea545b1880686e426b1acae7d26f0:

First, can you split this into 80- or 100-character lines? Then, a few notes:

1. 10-15% is pretty big. The bulk of this comes from replacing txid/vout with block/output indices. This has a heavy price: you cannot refer to unconfirmed transactions and the encoding is not reorg-safe.
2. More savings come from simply deleting data and grinding to recover it, which has a heavy CPU cost.
3. Also, you mention signature recovery. We should de-emphasize this. What we can say is that with Taproot key-spends, the public key for each input does not appear in the transaction; using pubkey recovery, we can frequently simulate this for legacy and segwit transaction.
4. Finally, "low bandwidth" should be expanded to say "low bandwidth, but with a lot of CPU power expendable on decoding".

Reply:
Should maybe also suggest that it's for applications where users can deliberately reduce the number of inputs/outputs in their transactions.

Reply:
Fixed, Thanks!